.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: ci
+
+on:
+  - push
+  - pull_request
```

```suggestion
  push:
    branches:
      - master
  - pull_request
```

To avoid 2 runs for each PR.

---

That makes sense, I am going to apply the suggestion 😄